src/device/device.hpp

```diff
@@ -28,6 +28,20 @@
 //
 
 
+/* Note about debug:
+ * To debug Device you can def the following :
+ * #define DEBUG_HWDEVICE
+ *   Activate debug mechanism:
+ *     - Add more trace
+ *     - All computation done by device are checked by default device.
+ *       Required IODUMMYCRYPT_HWDEVICE or IONOCRYPT_HWDEVICE for fully working
+ * #define IODUMMYCRYPT_HWDEVICE 1
+ *     - It assumes sensitive data encryption is is off on device side. a XOR with 0x55. This allow Ledger Class to make check on clear value
+ * #define IONOCRYPT_HWDEVICE 1
+ *     - It assumes sensitive data encryption is off on device side.
```

Comment:
This is only related to application option compilation. Application available to dowload/intall from Ledegr manger will never propose those options compiled in. 

Anyone who want to play with thoses option  for debugging have to get sources, get sdk, compile its version,  and install it with sdk-loader. So  deactivate strong encryption is always intentional.

Reply:
I was thinking, if someone steals a ledger, and if the device is flashable, then all a thief needs to do is flash a debug firmware, right ? Then the fact that the release firmware does not send the secret keys doesn't really matter in the first place. Intentional does not matter here since a thief would definitely do this intentionally.
Anyway, that's a separate thought, the patch is accepted.

Reply:
I propose to add a detailed note about that in the README of the application.
ok?